ruma-events/src/room/redaction.rs

```diff
@@ -55,9 +55,21 @@ pub struct RedactionEventStub {
     pub unsigned: UnsignedData,
 }
 
+// TODO is this possible or should we special case this in the macro
+/// A redacted redaction event.
+#[derive(Clone, Debug, Deserialize, Serialize, EventContent)]
+#[ruma_event(type = "m.room.redaction")]
+#[ruma_event(custom_redacted)]
+pub struct RedactedRedactionEventContent;
```

Comment:
Is this something that can happen or is it not possible to redact a redaction?

Reply:
I would expect this to be possible. It seems unlikely to come up in practice, but people could create spam that's hard to remove by putting it as the reason when redacting their own messages. I doubt any clients currently expose ways to redact redactions :D